apps/frontend/app/AdminChoachRequest/hooks/useCoachRequestGet.tsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+import { TrainerRequest } from "../../types";
+
+export const useCoachRequestGet = () => {
+    const [data, setData] = useState<TrainerRequest[]>([]);
+    
+    useEffect(() => {
+        async function fetchData() {
+        const response = await fetch('http://localhost:4444/trainer-requests/');
```

Comment:
in utils there is a file for connections and requests to the back end, place it there.